respeaker_ros/launch/sample_respeaker.launch

```diff
@@ -13,23 +13,30 @@
   <arg name="language" default="en-US"/>
   <!-- self cancellation -->
   <arg name="self_cancellation" default="true"/>
+  <!-- audio info topic name -->
+  <arg name="audio_info" default="audio_info"/>
```

Comment:
Setting `audio_info` to `"audio_info"` may change the default behavior.
In fact, `sample_respeaker.test` fails with this PR.[

Setting `audio_info` to `"false"` is correct?
```
<arg name="audio_info" default="false"/>
```

Reply:
Thanks for your comment!
I modified the test code's argument.
Ideally, the default is to output audio_info and the user does not need to give parameters such as samplerate.

Reply:
OK, I see.

I think it is good to publishing audio_info by default